### PR TITLE
Modified CartPage items title

### PR DESCRIPTION
**Pull request by winniepukki**

### CODE REVIEW

File: packages/scandipwa/src/route/CartPage/CartPage.container.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ export class CartPageContainer extends PureComponent {
         }
 
         if (items_qty !== prevItemsQty) {
-            const title = `${ items_qty || '0' } Items`;
+            const title = `${ items_qty || '0' } Item(s)`;
```
Comment (alfredsgenkins) [inline packages/scandipwa/src/route/CartPage/CartPage.container.js:131]: This string is not translated, please add translation to it!

**Review by alfredsgenkins — CHANGES_REQUESTED** [review]

**Comment by winniepukki** [comment]
Changes from [the following](https://github.com/winniepukki/scandipwa/tree/1843-cart-items-mobile-title) pull request don't get to here

**Comment by alfredsgenkins** [comment]
The issue was not resolved, why did you marked it this way?

**Comment by winniepukki** [comment]
> Changes from [the following](https://github.com/winniepukki/scandipwa/tree/1843-cart-items-mobile-title) pull request don't get to here

Closed after the following issue

**Pull request closed without merging**